### DISCUSSION TITLE
No clue about the error

### DISCUSSION THREAD

**Discussion by gauravchakrabarty** [cu-esiil-edu/2024-data-short-course#99] [Q&A]
@eculler @nquarder 
An error I received while working on the vegetation workbook. (Screen Shot attached)
![E1](https://github.com/cu-esiil-edu/2024-data-short-course/assets/8917893/848c02a9-f8b6-403e-860a-b71af65b374a)
![E1_contd](https://github.com/cu-esiil-edu/2024-data-short-course/assets/8917893/124162ee-48d5-4e3a-9109-2c5a1cc698da)
Can you please explain the reason why I was getting it?

**Comment by nquarder** [comment]
Hi @gauravchakrabarty - I will take a look at your code and get back to you soon with a reply. Sorry we are doing our Summit this week at ESIIL and are a bit backed up right now. Stay tuned! :) -NQ

**Reply by gauravchakrabarty** [reply]
Thanks, Sir !! 
I have also submitted the first badge assignment.
